client/js/views/GameView.vue

```diff
@@ -1108,7 +1108,9 @@ export default {
   color: #fff;
   width: 100vw;
   height: 100%;
-  background: linear-gradient(180deg, #6202ee 14.61%, #fd6222 100%), #c4c4c4;
```

Comment:
I feel like these should be css variables

Reply:
Yes but also they're being removed so screw it?

Reply:
@peterancourt we may just want to do a full overhaul of raw hex to css variables in the future. In which case just leave these for now and file a follow up.

Reply:
If we use something like that in the future then I definitely agree